frontend/src/store/modules/UserModule.ts

```diff
@@ -42,10 +42,10 @@ class UserModule extends VuexModule {
     }
 
     @Action
-    login({email, password} : ILoginPayload) {
+    async login({email, password} : ILoginPayload) {
         this.setIsLoading(true);
         this.setFailure("");
-        axios.post("http://localhost:9292/api/v1/users/login", {email, password})
+        await axios.post("http://localhost:9292/api/v1/users/login", {email, password})
             .then( response => {
                 const decoded : IUser = jwt.decode(response.data.token) as IUser;
                 decoded.username = response.data.username
```

Comment:
I quote you on dis "My code follows the style guidelines of this project" 
But wait. where is da semi colon ∂Ω∂œı√ƒéıœƒπçœı√÷ﬁß∂ç‹˛›éΩœıƒ˛Ωœıƒüé˛œΩ[ƒyzqo
Illuminati confirmed:eyes::eyes::eyes::eyes::eyes::eyes::eyes:
```suggestion
                decoded.username = response.data.username;
```

Reply:
På axios saken så forstätter tekniskt sett raden efter, hence inget ;